src/Dapr.AspNetCore/DaprBulkAppResponse.cs

```diff
@@ -0,0 +1,44 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2021 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Collections.Generic;
+
+namespace Dapr.AspNetCore
+{
+    /// <summary>
+    /// Response from the application containing status for each entry in the bulk message.
+    /// It is posted by the bulk subscribe handler.
+    /// </summary>
+    public class DaprBulkAppResponse
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DaprBulkAppResponse"/> class.
+        /// </summary>
+        public DaprBulkAppResponse() {
+        }
+        
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DaprBulkAppResponse"/> class.
+        /// </summary>
+        /// <param name="statuses">List of statuses.</param>
+        public DaprBulkAppResponse(List<DaprBulkAppResponseEntry> statuses)
+        {
+            this.statuses = statuses;
+        }
+        
+        /// <summary>
+        /// List of statuses.
+        /// </summary>
+        public List<DaprBulkAppResponseEntry> statuses { get; set; }
```

Comment:
Generally, publicly accessible fields should be capitalized also this should be Entries or something like that since you need the individual entry to get the status.

Reply:
Fixed